### DISCUSSION TITLE
Add support for `qmd` file (from `Quarto` publishing)?

### DISCUSSION THREAD

**Discussion by hopezh** [hedyhli/outline.nvim#52] [Help] [answered]
`.qmd` file is a kind of `markdown` file format developed by [Quarto publishing system](https://quarto.org/).  

May I ask how to add the support for `qmd` file so that I can check all the markdown headings, as show below for a `.md` file? Thanks. 

<img width="1443" alt="Screenshot 2024-02-04 at 22 45 55" src="https://github.com/hedyhli/outline.nvim/assets/23435810/8e70adfd-3f57-4129-9771-737aeb4a9fa2">

**Comment by lahvak** [comment]
Please, and the same with rmarkdown.  Perhaps a way to specify that one file type should be handled as another would suffice.

**Comment by hedyhli** [comment, marked as answer]
This feature has been added in #58, thanks for the suggestion!

You can now add:
```lua
providers = {
  -- ...
  markdown = {
    filetypes = { 'markdown', 'rmarkdown', 'quarto' },
  },
}
```

to your config, where `quarto`/`rmarkdown` depends on the `ft` value you have set for those filetypes (see `set ft?`).
